pkl-core/src/main/java/org/pkl/core/util/GlobResolver.java

```diff
@@ -488,7 +489,14 @@ public static List<ResolvedGlobElement> resolveGlob(
         }
         return result;
       }
-      var baseUri = enclosingModuleKey.resolveUri(enclosingUri, URI.create(basePath));
+      URI baseUri;
+      try {
+        baseUri = IoUtils.resolve(securityManager, enclosingModuleKey, URI.create(basePath));
+      } catch (URISyntaxException e) {
+        // assertion: this is only thrown if the pattern starts with a triple-dot import.
+        // the language will throw an error if glob imports is combined with triple-dots.
```

Comment:
In that case, isn't this unreachable code?

Reply:
I think I'll leave this as-is; not 100% sure this is unreachable.